test/src/main/java/no/nordicsemi/andorid/ble/test/client/view/ClientScreen.kt

```diff
@@ -0,0 +1,42 @@
+package no.nordicsemi.andorid.ble.test.client.view
+
+import android.os.Build
+import androidx.annotation.RequiresApi
+import androidx.compose.foundation.layout.Column
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.res.stringResource
+import androidx.hilt.navigation.compose.hiltViewModel
+import no.nordicsemi.andorid.ble.test.R
+import no.nordicsemi.andorid.ble.test.client.viewmodel.ClientViewModel
+import no.nordicsemi.andorid.ble.test.server.view.LoadingView
+import no.nordicsemi.andorid.ble.test.server.view.ResultView
+import no.nordicsemi.android.ble.ktx.state.ConnectionState
+import no.nordicsemi.android.common.permission.RequireBluetooth
+import no.nordicsemi.android.common.permission.RequireLocation
+import no.nordicsemi.android.common.theme.view.NordicAppBar
+
+@OptIn(ExperimentalMaterial3Api::class)
+@RequiresApi(Build.VERSION_CODES.O)
+@Composable
+fun ClientScreen() {
+    Column {
+        NordicAppBar(text = stringResource(id = R.string.scanner))
+        RequireBluetooth {
+            RequireLocation {
+                val clientViewModel: ClientViewModel = hiltViewModel()
+                val clientViewState by clientViewModel.clientViewState.collectAsState()
```

Comment:
Roshan asked to use collectAsStateWithLifecycle() everywhere. 
Here is an article why: https://medium.com/androiddevelopers/consuming-flows-safely-in-jetpack-compose-cde014d0d5a3.
I will also change it every application I have.